tests/test_converter.py

```diff
@@ -780,7 +794,7 @@ def test_map_wires(self, recorder):
         qc = QuantumCircuit(1)
         qc_wires = [hash(q) for q in qc.qubits]
 
-        assert map_wires(wires, qc_wires) == {0: hash(qc.qubits[0])}
+        assert map_wires(qc_wires=wires, wires=qc_wires) == {0: hash(qc.qubits[0])}
```

Comment:
Why this is changed?

Reply:
`pylint` was complaining about the reversal of arguments.